roiextractors/extractors/caiman/caimansegmentationextractor.py

```diff
@@ -79,14 +79,18 @@ def roi_locations(self):
         return roi_locations
 
     @staticmethod
-    def write_segmentation(segmentation_object, savepath, **kwargs):
-        plane_no = kwargs.get('plane_no', 0)
-        filename = os.path.basename(savepath)
-        savepath_folder = os.path.join(os.path.dirname(savepath), f'Plane_{plane_no}')
-        savepath = os.path.join(savepath_folder, filename)
-        if savepath.split('.')[-1]!='hdf5':
+    def write_segmentation(segmentation_object, save_path, plane_num=0):
+        if save_path.split('.')[-1]!='hdf5':
```

Comment:
can we use pathlib?

Reply:
implemented https://github.com/catalystneuro/roiextractors/pull/37/commits/d21c64caa880457f47b22e65b36422c18f2b99c0